docs/advanced/DAP.md

```diff
@@ -1,4 +1,4 @@
-# CMSIS-DAP and DAPLink
+# DAPLink
 
 
 ## DAPLink
```

Comment:
Please delete the second heading, so we don't have two headings in a row that say the same thing